### ISSUE TITLE
CLOB and BLOB datatype are not supported

### ISSUE THREAD

**Issue by harsh288** [rsim/oracle-enhanced#2211]
Hello Everyone,

I'm using fluentd and using fluent SQL plugin along with oracle_enhanced adapter, but the datatype clob and blob are not working, please help.

```
<match MIOLogMessage>
	@type sql
    flush_interval 10
	host localhost
	database GOAORA
	port 1521
	adapter oracle_enhanced
	username admin
	password admin
	<table>
		table logmessage
		column_mapping 'miologid:miologid,timestamputc:timestamputc,messageid:messageid'
	</table>
	flush_interval 1s
</match>
```

**Comment by akostadinov** [comment]
seems like a duplicate of #2216

**Comment by harsh288** [comment]
@akostadinov 

Still there is no support on this ??

**Comment by akostadinov** [comment]
@harsh288 , I don't use fluentd and can't tell. I'm trying to submit the hacks from our repo here one by one. The original issue seems to be closed and I didn't see the described issue here when running oracle-enhanced test suite but again, not using fluentd so I don't know what is it doing.

Do you see the issue with the latest commit of oracle-enhanced? If you can test that, it may help. I see some strange behavior with lobs in master as well I still have a few hacks related to lobs in my repo that I need to upstream (if they are still relevant). So checking how latest commit works for you may help. Also posting some logs of the SQL queries leading to the issue may also be very useful.

**Comment by harsh288** [comment]
@akostadinov Yeah, I haven't tried the latest package, just checking if it's fixed in the latest pkg so that I can install it.. will update you, thanks !!
